dns/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
```

Comment:
I like ubuntu-slim because i can apt-get. If you're just looking for a substrate for a static go binary, scratch is tiny (https://docs.docker.com/engine/userguide/eng-image/baseimages/), busybox the next best.